packages/carbon-graphs/package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@cerner/carbon-graphs",
+  "main": "lib/Graphs.js",
+  "private": true,
+  "version": "0.0.0",
+  "description": "A graphing library built using d3 based on Cerner design standards",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-graphs.git",
+    "directory": "packages/carbon-graphs"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "carbon-graphs",
+    "carbon",
+    "graphs",
+    "UI",
+    "d3",
+    "d3-module",
+    "graphing"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-graphs/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-graphs#readme",
+  "files": [
+    "dist",
+    "lib",
+    "README.md"
+  ],
+  "prettier": {
+    "arrowParens": "always",
+    "bracketSpacing": true,
+    "semi": true,
+    "tabWidth": 4,
+    "trailingComma": "none",
+    "overrides": [
+      {
+        "files": "package.json",
+        "options": {
+          "tabWidth": 2,
+          "printWidth": 250
+        }
+      }
+    ]
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "autoprefixer": "^9.8.4",
+    "classnames": "^2.2.5",
+    "d3": "^5.16.0",
+    "jasmine-core": "^3.5.0",
+    "prop-types": "^15.5.8",
+    "terra-theme-context": "^1.0.0"
+  },
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5"
+  },
```

Comment:
Do not add react to this package, nor should you be adding terra-theme-context.

Reply:
https://github.com/cerner/terra-graphs/pull/5/commits/a331e749bc1cbe1d46395138900962a2d11a1425